gr-qtgui/lib/sink_c_impl.cc

```diff
@@ -187,41 +163,41 @@ void sink_c_impl::set_frequency_range(const double centerfreq, const double band
 {
     d_center_freq = centerfreq;
     d_bandwidth = bandwidth;
-    d_main_gui->setFrequencyRange(d_center_freq, -d_bandwidth, d_bandwidth);
+    d_main_gui.setFrequencyRange(d_center_freq, -d_bandwidth, d_bandwidth);
 }
 
 void sink_c_impl::set_fft_power_db(double min, double max)
 {
-    d_main_gui->setFrequencyAxis(min, max);
+    d_main_gui.setFrequencyAxis(min, max);
 }
 
-void sink_c_impl::enable_rf_freq(bool en) { d_main_gui->enableRFFreq(en); }
+void sink_c_impl::enable_rf_freq(bool en) { d_main_gui.enableRFFreq(en); }
 
 /*
 void
 sink_c_impl::set_time_domain_axis(double min, double max)
 {
-  d_main_gui->setTimeDomainAxis(min, max);
+  d_main_gui.setTimeDomainAxis(min, max);
 }
 
 void
 sink_c_impl::set_constellation_axis(double xmin, double xmax,
                                     double ymin, double ymax)
 {
-  d_main_gui->setConstellationAxis(xmin, xmax, ymin, ymax);
+  d_main_gui.setConstellationAxis(xmin, xmax, ymin, ymax);
 }
 
 void
 sink_c_impl::set_constellation_pen_size(int size)
 {
-  d_main_gui->setConstellationPenSize(size);
+  d_main_gui.setConstellationPenSize(size);
 }
 */
```

Comment:
Would it make sense to remove this section? For now, it's dead code.

Reply:
You mean because they're commented out in the public header file?

I would prefer in a separate commit, but either works for me.

Reply:
A separate commit would be reasonable. I just stumbled over. Obviously, this could is prone to become stale. I'd say, either we use it or remove it.

Reply:
Added a commit deleting this commented out code.